### DISCUSSION TITLE
Reference object of another folder

### DISCUSSION THREAD

**Discussion by Julia-SOLIGNAC** [pimcore/pimcore#12540] [Q&A / Support]
Hello :)

I have a question :
Is it somehow possible to reference objects or asset from a folder to another folder, like an anchor or a pointer to it ?

Because i have differents folder to split products by brand, but some products can have two brands. 
I cannot duplicate product, so i'm looking for an alternative system.

Is there something like this in Pimcore ?

I'm also looking on the pimcore customs view, to avoid sorting by folder. But for now i'm just wondering if reference to objects or assets exist.

**Comment by BlackbitDevs** [comment]
Currently this does not exist. A solution was proposed in #4419 but not yet implemented. 
You can have a look at https://github.com/elements-at/AlternateObjectTrees but currently this only supports to set up different trees but at the moment there is no way to display the same object under 2 parent nodes. But of course this would be a cool feature, so  we are looking for someone who would pay for the implementation...
 
Currently we implement this with relations. In your example you could have "Product 1" in `/Brands/Brand A` and have a relational field in the Product class for "additional brands". To display all brands of a product, there could be a calculated value field which lists the main brand (= parent object) + additional brands from the relation. Of course it is not really the same because you do not see in the tree which products belong to brand `/Brands/Brand B` when some of the products are actually assigned to `/Brands/Brand A` (and only related in the "additional brands" to `/Brands/Brand A` - you would only see it if you open the object.
